### PR TITLE
Added a git video tutorial in videos sections

### PR DESCRIPTION
**Pull request by choudharyanurag**
# Quality Standards

Note that we can help you achieve those standards, just try your best and be brave.
We'll guide you to the best of our abilities.

To be on the list, it would be **nice** if entries adhere to these quality standards:

- It should take less than 20 sec to find what is the project, how to install it and how to use it.
- Generally useful to the community.
- A project on GitHub with a well documented `README.md` file and plenty of examples is considered high quality.
- Clearly stating if an entry is related to (Linux) containers and not to Docker. There is an [awesome list](https://github.com/Friz-zy/awesome-linux-containers) for that.
- Clearly stating "what is it" i.e. which category it belongs to.
- Clearly stating "what is it for" i.e. mention a real problem it solves (even a small one). Make it clear for the next person.
- If it is a **WIP** (work in progress, not safe for production), please mention it. (Remember the time before Docker 1.0 ? ;-) )
- Always put the link to the GitHub project instead of the website!

To be on the list, the project **must** have:

- How to setup/install the project
- How to use the project (examples)

If your PR is not merged, we will tell you why so that you may be able to improve it.
But usually, we are pretty relaxed people, so just come and say hi, we'll figure something out together.

# Rules for Pull Request

- Each item should be limited to one link, no duplicates, no redirection (careful with `http` vs `https`!)
- The link should be the name of the package or project or website
- Description should be clear and concise (read it out loud to be sure)
- Description should follow the link, on the same line
- Entries are listed alphabetically, please respect the order
- If you want to add more than one link, please don't do all PR on the exact same line, it usually results in conflicts and your PR cannot be automatically merged...

Please contribute links to packages/projects you have used or are familiar with. This will help ensure high-quality entries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -748,6 +748,7 @@ Services to securely store your Docker images.
 - [Performance Analysis of Docker - Jeremy Eder](https://www.youtube.com/watch?v=6f2E6PKYb0w) (1:36:58)
 - [Scalable Microservices with Kubernetes](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615) Free Udacity course
 - [State of containers: a debate with CoreOS, VMware and Google](https://www.youtube.com/watch?v=IiITP3yIRd8) (27:38)
+- [Docker Primer to Docker Compose](https://www.youtube.com/watch?v=G-s2GXGAjTk) (1:56:45)
```
Comment (veggiemonk) [inline README.md:751]: Nice! Could you put it in alphabetical order as described in the contribution guidelines ?

**Review by veggiemonk — APPROVED** [review]
Nice, just ordering the entry and it's good to go

**Review by veggiemonk — CHANGES_REQUESTED** [review]
Just one little thing to order

File: README.md (hunk 1 above)
Comment (choudharyanurag) [reply README.md]: Done, just now!

**Review by mash1t — APPROVED** [review]

**Review by veggiemonk — APPROVED** [review]

**Pull request merged**